notebooks/tutorial/generate_danish_model_from_butler.ipynb

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
<p>LGTM, just a suggestion to merge these three cells. If you initialize the butler as Butler('main', collection=blabla, instrument=blabla) you don't have to pass that later and things look cleaner. Just a suggestion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/65/discussion/#comment-2516417653'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>